CODE_OF_CONDUCT.md

```diff
@@ -1,27 +1,90 @@
 ---
-title: RECON Contributor Code of Conduct
+title: RECON Code of Conduct
 ---
 
-As contributors and maintainers of this project, we pledge to respect all people who 
-contribute through reporting issues, posting feature requests, updating documentation,
-submitting pull requests or patches, and other activities.
```

Comment:
- Contribute to RECON training activities or deployments

Reply:
You're commenting on deleted text here. I believe this is covered in the "Scope" paragraph below: 

> This Code of Conduct applies within all RECON community spaces